### PR TITLE
netbsd login api addition

### PR DESCRIPTION
**Pull request by devnexen**

### CODE REVIEW

**Comment by JohnTitor** [comment]
Seems highfive fails to assign someone, r? @JohnTitor

File: src/unix/bsd/netbsdlike/netbsd/mod.rs [hunk 1]
```diff
@@ -2294,6 +2294,9 @@ extern "C" {
         denominator: ::intmax_t,
         precision: ::size_t,
     ) -> *mut ::c_char;
+    pub fn login(ut: *mut utmp);
+    pub fn logout(line: *mut ::c_char);
+    pub fn logwtmp(line: *mut ::c_char, name: *mut ::c_char, host: *mut ::c_char);
```
Comment (JohnTitor) [inline src/unix/bsd/netbsdlike/netbsd/mod.rs:2299]: Seems [they all are `*const`](https://github.com/NetBSD/src/blob/05082e19134c05f2f4b6eca73223cdc6b5ab09bf/include/util.h#L88-L94), aren't they?

Comment (devnexen) [reply src/unix/bsd/netbsdlike/netbsd/mod.rs:2299]: yes went too fast on this one.

Comment (JohnTitor) [reply src/unix/bsd/netbsdlike/netbsd/mod.rs]: You forgot to update `login`.

Comment (devnexen) [reply src/unix/bsd/netbsdlike/netbsd/mod.rs]: login is correct tough ?

Comment (devnexen) [reply src/unix/bsd/netbsdlike/netbsd/mod.rs]: ah right I see the signature of login had been changed in the meantime but in the current release it s "mutable"

**Review by JohnTitor — APPROVED** [review]

**Comment by JohnTitor** [comment]
Thanks :)
@bors r+

**Pull request merged**